### PR TITLE
DEV: replace survey 'Y' with booleanAttributes

### PR DESCRIPTION
**Pull request by TonisOrmisson**

### CODE REVIEW

File: application/helpers/remotecontrol/remotecontrol_handle.php [hunk 1]
```diff
@@ -2618,11 +2618,11 @@ public function update_response($sSessionKey, $iSurveyID, $aResponseData)
         if (is_null($oSurvey)) {
             return 'Error: Invalid survey ID';
         }
-        if ($oSurvey->getAttribute('active') !== 'Y') {
+        if ($oSurvey->isActive) {
```
Comment (tammoterhark) [inline application/helpers/remotecontrol/remotecontrol_handle.php]: isNotActive?

File: application/helpers/remotecontrol/remotecontrol_handle.php [hunk 2]
```diff
@@ -2618,11 +2618,11 @@ public function update_response($sSessionKey, $iSurveyID, $aResponseData)
...
             return 'Error: Survey is not active.';
         }
 
-        if ($oSurvey->getAttribute('alloweditaftercompletion') !== 'Y') {
+        if ($oSurvey->isAllowEditAfterCompletion) {
```
Comment (tammoterhark) [inline application/helpers/remotecontrol/remotecontrol_handle.php]: not?

Comment (TonisOrmisson) [reply application/helpers/remotecontrol/remotecontrol_handle.php]: fix coming, sry :)

File: application/helpers/remotecontrol/remotecontrol_handle.php (hunk 1 above)
Comment (Shnoulle) [reply application/helpers/remotecontrol/remotecontrol_handle.php]: or `!$oSurvey->isActive` :)

**Pull request merged**